lib/bytes_by_prefix.c

```diff
@@ -62,12 +62,16 @@
 #define B_MULT	BSIZE		/* block size */
 #elif sgi
 #define B_MULT	BBSIZE		/* block size */
-#elif defined(__linux__) || defined(__sun) || defined(__hpux)
+#elif (defined(__linux__) || defined(__sun) || defined(__hpux)) && defined(DEV_BSIZE)
 #define B_MULT	DEV_BSIZE	/* block size */
```

Comment:
We should get rid of the __sun, __hpux, sgi and _AIX ifdefs first (in a separate cleanup patch).